contrib/babelfishpg_tsql/src/pltsql_coerce.c

```diff
@@ -1531,24 +1531,24 @@ PG_FUNCTION_INFO_V1(ftrunci8);
 Datum
 ftrunci8(PG_FUNCTION_ARGS)
 {
-	float4		num = PG_GETARG_FLOAT4(0);
+	float4		num = PG_GETARG_FLOAT4(0), arg = 0;
```

Comment:
Seems like we are using another variable to avoid warnings, if we are sure that warning is false positive then we should use inline suppression instead of making code changes.